docs/application/dotnet/guides/tizen.circularUI/ConfirmationPopup.md

```diff
@@ -0,0 +1,84 @@
+---
+uid: Tizen.Wearable.CircularUI.doc.ConfirmationPopup
+summary: ConfirmationPopup control guide
+---
+
+# ConfirmationPopup
+`ConfirmationPopup` represents The Tizen Wearable EFL two button popup style. The `ConfirmationPopup` is used to select one of the two options such as cancel or done, and share or delete.
+
+*Since `ConfirmationPopup` does not adhere as a control, this popup cannot be set in the XAML file.*
+
+![confirmation_popup1](data/confirmation_popup1.png)
+
+## Create ConfirmationPopup
+`ConfirmationPopup.Title` property sets the title of the popup. You can set the `Content` property with Layouts such as `StackLayout` or `ScrollView`.
+`ConfirmationPopup.FirstButton` property sets left side button. `ConfirmationPopup.SecondButton`property sets right side button. You can set` FirstButton` and `SecondButton` using `MenuItem`. You should add code at `Command` or `Clicked` event handler for controlling `MenuItem` clicked.
```

Comment:
You should add code at `Command` or `Clicked` event handler for controlling `MenuItem` clicked.
Please change:
You must add code at `Command` or `Clicked` event handler for controlling `MenuItem` clicked.